ispconfig.php

```diff
@@ -1476,12 +1524,13 @@ private function updateAccountCount($module_row)
      * @param mixed $hostname
      * @param mixed $username
      * @param mixed $use_ssl
+     * @param mixed $port
      * @return bool True if the connection is valid, false otherwise
      */
-    public function validateConnection($password, $hostname, $username, $use_ssl, &$account_count)
+    public function validateConnection($password, $hostname, $username, $use_ssl, &$account_count, $port)
```

Comment:
This is a public method, so it wouldn't be backward-compatible to require a new argument. You should set a default value for it